packages/core/theme-module/theme/pages/MyAccount.vue

```diff
@@ -37,18 +37,24 @@
         <SfContentPage title="My reviews">
           <MyReviews />
         </SfContentPage>
+        <SfContentPage title="Order details">
+          <OrderDetails v-if="entityId" :orderId="entityId" />
```

Comment:
if entityID is taken from URL it should be a part of `<OrderDetails>` component imho

Reply:
I think it's better to provide it through props as it gives better control, but we can discuss about it

Reply:
what do you mean by better control?

Reply:
By providing entityId as a prop you can get it on a way you define (or URL, or cookie, or cache). Simply: it's open for extensions. If I'll define entityId directly in OrderDetails I'm forced to get entityId from URL only.

Reply:
no, you can still change it in the exactly same way you're changing `entityId` prop

Reply:
Imho what I want to discuss here is do we pass always via props or always use composable inside for all the pages, I'm ok with both just keep it consistant

Reply:
as we agreed - I removed that prop